src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/request/MatchPrisonersRequest.kt

```diff
@@ -18,7 +18,7 @@ data class MatchPrisonersRequest(
   val lastName: String,
 
   @Schema(
-    description = "Date of birth (If absent then, will not perform name and date of birth search)",
+    description = "Date of birth",
     example = "1971-02-01"
   )
   @field:NotNull
```

Comment:
We need to leave DoB as nullable as this object is used by ArrivalService to make query without DoB. 
@field:NotNull will return bad request for this specific endpoint and prevent us from creating one more data object for PrisonerSearchService.